Dockerfile

```diff
@@ -12,3 +12,7 @@ RUN apk update && apk upgrade && \
 
 ADD package.json $HOME
 RUN npm install --silent
+
+ADD . $HOME
+
+CMD npm start
```

Comment:
This pulls everything into the image on the initial build. We still mount the current directory over `$HOME` in our compose file so this doesn't affect us during dev at all.

Reply:
Won't this potentially include secrets in the image, `.git` directory, etc? Maybe we should add a bundling step that copies files off to a `dist/` directory and copy that directory into the image?

If we follow a similar approach with `.secrets/` in this repo like I did in windbreaker-io/windbreaker-hooks#5, this would be problematic.

Reply:
Maybe it's best not to store the secrets in a folder inside of the project itself. Thoughts? Also I still think that the secrets folder needs to be configurable.

Reply:
I don't think we should be storing secrets within our repos. It doesn't make sense to do so given our current goals for this project. Ideally, secrets would need to be dropped in or retrieved from a secrets management service either at deploy time or during runtime (respectively).

@Druotic Since we _are_ early on in dev, I can setup the `.dockerignore` to never add the `.secrets` file and also setup temporarily setup secret mounting in our `docker-compose.yml`